scipy/signal/__init__.py

```diff
@@ -343,14 +345,19 @@
                       'nuttall', 'blackmanharris', 'flattop', 'bartlett',
                       'barthann', 'hamming', 'kaiser', 'gaussian',
                       'general_gaussian', 'chebwin', 'cosine',
-                      'hann', 'exponential', 'tukey')
+                      'hann', 'exponential', 'tukey', 'taylor', 'lanczos',
+                      'dpss')
 
 
 def deco(name):
     f = getattr(windows, name)
     # Add deprecation to docstring
 
     def wrapped(*args, **kwargs):
+        warnings.warn(f"Importing {name} from 'scipy.signal' is deprecated "
+                      "as of SciPy 1.11.0 and will raise an error in SciPy "
+                      "1.13.0.",
```

Comment:
IMO:
* we should provide clear instructions how to resolve/avoid the warning
* there's no point in mentioning when it was deprecated (which, arguably here was for 1.1 already anyway)

```suggestion
        warnings.warn(f"Importing {name} from 'scipy.signal' is deprecated "
                      "and will raise an error in SciPy 1.13.0. This function "
                      "should be imported from 'scipy.signal.windows'",
```

Reply:
Updated